CalibCalorimetry/EcalPedestalOffsets/src/testChannel.cc

```diff
@@ -85,8 +86,7 @@ void testChannel::analyze(edm::Event const &event, edm::EventSetup const &eventS
 
   // get the digis
   // (one digi for each crystal)
-  edm::Handle<EBDigiCollection> pDigis;
-  event.getByLabel(m_digiProducer, pDigis);
+  const edm::Handle<EBDigiCollection> &pDigis = event.getHandle(m_digiProducerToken);
   if (!pDigis.isValid()) {
     edm::LogError("testChannel") << "Error! can't get the product " << m_digiCollection.c_str();
```

Comment:
One could avoid propagating the string as class member by wrting something like:
```suggestion
    edm::LogError("testChannel") << "Error! can't get the product pDigis";
```
This was already wrong, in any case, because it should have been `m_digiProducer` and not `m_digiCollection`: `m_digiCollection` is not used anywhere, probably a legacy of some older version of the code